jvm/src/main/java/com/microsoft/bond/protocol/BinaryStreamReader.java

```diff
@@ -0,0 +1,81 @@
+package com.microsoft.bond.protocol;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+/**
+ * Responsible for reading Bond primitive data types from an input stream.
+ * This class encapsulates little-endian to big-endian conversion.
```

Comment:
Updated comment.

---
In reply to: [116112783](https://github.com/tstein/bond/pull/4#discussion_r116112783) [](ancestors = 116112783)